lib/SILOptimizer/Mandatory/TFDeabstraction.cpp

```diff
@@ -2038,7 +2042,12 @@ formGraphOp(SILTensorOpInfo &opInfo,
     auto attrName = operandClass.first.str() +
        SILTensorOpInfo::getOperandClassSuffix(operandClass.second);
 
-    auto constValue = it->second.cloneInto(allocator);
+    // Get the constant, ignoring struct wrappers.
+    auto constValue = it->second.lookThroughSingleElementAggregates();
```

Comment:
At this point, it is preemptively copying the value in, allowing the code right below to validate it.  I've taken the approach of being simple instead of being perfectly precise: intermediate struct wrappers can be introduced for a bunch of reasons, and I don't see a reason to have a tight coupling with the stdlib implementation details.

Array and aggregates are modeled as separate concepts, so the lookThroughSingleElementAggregates helper does not dig through single element arrays (I agree that this would be super problematic).